cli/config.go

```diff
@@ -1,36 +1,84 @@
 package cli
 
 import (
-	"errors"
 	"fmt"
-	"os"
 
+	"github.com/MakeNowJust/heredoc"
 	"github.com/odpf/compass/internal/client"
 	"github.com/odpf/compass/internal/server"
 	esStore "github.com/odpf/compass/internal/store/elasticsearch"
 	"github.com/odpf/compass/internal/store/postgres"
 	"github.com/odpf/compass/pkg/metrics"
 	"github.com/odpf/compass/pkg/statsd"
-	"github.com/odpf/salt/config"
+	"github.com/odpf/salt/cmdx"
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v2"
 )
 
 const configFlag = "config"
 
-func cmdShowConfigs() *cobra.Command {
+var cliConfig *Config
+
+func configCommand() *cobra.Command {
```

Comment:
Do we have a single file for client and server configs? I see we are loading client configurations but the removed code was reading server configurations. Although I believe instead of managing two files, we could have used one for both.

Reply:
Yes we have a single file for both